packages/kv-store/src/Redis/PhpRedisClient.php

```diff
@@ -99,7 +99,7 @@ public function command(Stringable|string $command, Stringable|string ...$argume
     public function set(Stringable|string $key, mixed $value, null|Duration|DateTimeInterface $expiration = null): void
     {
         if ($expiration instanceof DateTimeInterface) {
-            $expiration = DateTime::now()->since($expiration);
+            $expiration = $expiration->since(DateTime::now());
```

Comment:
Is `since` not absolute? If it is, I'd prefer like it was before

Reply:
Nope, it goes into negative unfortunately, that's the reason behind this change.